Kickstarter-iOS/Views/Controllers/ChangeEmailViewController.swift

```diff
@@ -32,11 +31,7 @@ internal final class ChangeEmailViewController: UIViewController {
   override func viewDidLoad() {
     super.viewDidLoad()
 
-    guard let messageBannerViewController = self.children.first as? MessageBannerViewController else {
-      fatalError("Couldn't instantiate MessageBannerViewController")
-    }
-    self.messageBannerViewController = messageBannerViewController
-    self.messageBannerViewController.delegate = self
+    self.messageBannerViewController = configureMessageBannerViewController(on: self)
```

Comment:
`configureMessageBannerViewController` should probably be `self.configureMessageBannerViewController` in the places where it's used 😬